src/docfx/lib/log/CustomRule.cs

```diff
@@ -23,6 +23,9 @@ internal class CustomRule
         [JsonConverter(typeof(OneOrManyConverter))]
         public string[] Exclude { get; private set; } = Array.Empty<string>();
 
+        [JsonConverter(typeof(OneOrManyConverter))]
+        public string[] ContentTypes { get; private set; } = Array.Empty<string>();
```

Comment:
This is actually called page type. Content type has a different meaning right now in docfx.

```suggestion
        public string[] PageTypes { get; private set; } = Array.Empty<string>();
```

Reply:
as talked, keep the naming since UI use it too